src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/containers/EntityFluidContainer.java

```diff
@@ -0,0 +1,15 @@
+package io.github.fablabsmc.fablabs.api.fluidvolume.v1.containers;
+
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api.FluidContainer;
+
+import net.minecraft.util.math.Direction;
+
+/**
+ * an entity that carries fluids.
+ */
+public interface EntityFluidContainer {
+	/**
+	 * @return the fluid container for this entity
+	 */
+	FluidContainer getContainer(Direction face);
```

Comment:
Is `Direction` a very usable concept on entities, since `Direction` is absolute cardinal and entities rotate? This is kinda similar to having chest/hopper minecarts implement `Inventory`; you can just access it from any face because trying to relativize `Direction`s is difficult.

Reply:
I have the direction, so might as well, you never know

Reply:
That's fair. It doesn't have to be used.

Reply:
Oh I forgot to mention as well, Entity Hitboxes are axis aligned anyways, so directions do make sense